clang/test/SemaHLSL/BuiltIns/rcp-errors.hlsl

```diff
@@ -0,0 +1,27 @@
+
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -verify -verify-ignore-unexpected
```

Comment:
We have all those intrinsic error check tests which looks the same.
Is it possible to make the builtin name as a macro and just add more run lines for each new builtin?

Something like
```

float test_too_few_arg() {
  return TEST_NAME();
  // expected-error@-1 {{too few arguments to function call, expected 1, have 0}}
}
```
and add 
```

-DTEST_NAME=__builtin_hlsl_elementwise_rcp 
```
to the RUN line

Reply:
not a bad idea, it shares the same sema checks as frac will try and get back to you.

Reply:
While it's certainly worth exploring the options here I worry that trying to deduplicate these too aggressively could make the tests a lot harder to understand.